Neos.Flow/Classes/Core/Booting/Scripts.php

```diff
@@ -249,16 +259,17 @@ public static function initializeSystemLogger(Bootstrap $bootstrap): void
      * Initialize the exception storage
      *
      * @param Bootstrap $bootstrap
-     * @param array $settings The Neos.Flow settings
      * @return ThrowableStorageInterface
      * @throws FlowException
      * @throws InvalidConfigurationTypeException
      */
-    protected static function initializeExceptionStorage(Bootstrap $bootstrap, array $settings): ThrowableStorageInterface
+    protected static function initializeExceptionStorage(Bootstrap $bootstrap): ThrowableStorageInterface
     {
+        $configurationManager = $bootstrap->getEarlyInstance(ConfigurationManager::class);
+        $settings = $configurationManager->getConfiguration(ConfigurationManager::CONFIGURATION_TYPE_SETTINGS, 'Neos.Flow');
```

Comment:
Why is this duplicated here from https://github.com/neos/flow-development-collection/pull/2449/files#diff-1da1ee0f3e18bd7b4ffbe3603640de236ca4dfd088f6414364536af0813d7438R244?